readme.md

```diff
@@ -16,14 +16,16 @@
 
 ### Laravel
 - [Laravel 5.1 Official Documentation](http://laravel.com/docs/5.1)
-- [Laravel PHP Components](https://github.com/illuminate)
+- [Laravel: From Apprentice To Artisan (Book)](https://leanpub.com/laravel)
 - [Awesome Laravel](https://github.com/chiraggude/awesome-laravel)
 - [Laravel 5 Essentials (Book)](https://www.packtpub.com/web-development/laravel-5-essentials)
 - [Learning Laravel’s Eloquent (Book)](https://www.packtpub.com/web-development/learning-laravel%E2%80%99s-eloquent)
 - [Mastering Laravel (Book)](https://www.packtpub.com/web-development/mastering-laravel)
+- [Laravel Testing Decoded (Book)](https://leanpub.com/laravel-testing-decoded)
```

Comment:
Hai cuốn [Laravel: From Apprentice To Artisan (Book)](https://leanpub.com/laravel) và [Laravel Testing Decoded (Book)](https://leanpub.com/laravel-testing-decoded) đều cổ quá, đều từ thời Laravel 4 :-s 
Cái From Apprentice To Artisan thì viết về core Laravel nên add thêm vào chắc cũng ổn, nhưng cái Laravel Testing Decoded anh nghĩ không cần em ạ :D Mình chưa yêu cầu cao phần viết test, không cần tốn thời gian vào đọc quyển đó =)) Dành tìm hiểu các phần khác thì hơn :D